docs/rfcs/text/YYYYmmDD-template.md

```diff
@@ -0,0 +1,22 @@
+- Feature Name: (fill me in with a unique ident, my_awesome_feature)
```

Comment:
Maybe we can organize rfcs directory like this
```
$ tree
.
├── 20220601-awesome-feature.md
└── resources/
```
It's more easy to refer images in `resources` directory.

Reply:
> It seems hard to look up the docs if the file is named after date. Should we name these files according to its content?

The date is one part of file name. It should follow by feature name, like `20220616-prom_remote_storage.md`

> What does the .gitkeep do?

Empty dir won't be tracked by git. That is a placeholder.

Reply:
👍